### ISSUE TITLE
Mounting local Maven Repo

### ISSUE THREAD

**Issue by deelthor** [arun-gupta/docker-java-multistage#2]
First thanks for this example. 
I was just wondering whether you see a possibility to specify an existing local Maven repository which could be used for the build. This way not all artifacts have to be downloaded  every time during a build (considering one would have a application with more dependencies).
Thanks,
Thorsten

**Comment by arun-gupta** [comment]
You mean mounting a volume referring to your local `.m2`?

**Comment by deelthor** [comment]
Basically yes. :)

**Comment by pomverte** [comment]
Yes, that would be helpful since the dependencies might not change that often except for SNAPSHOT
